.github/workflows/live-deploy.yml

```diff
@@ -0,0 +1,30 @@
+# An Action to build Web Arcs and deploy to https://live.arcs.dev
+name: live-deploy
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run Typescript Tests
+        run: |
```

Comment:
Is `|` a typo?

Reply:
Nope! I need it to specify that the next two lines need to be run as a script.